.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "files.readonlyInclude": {
+    "features/*.yml.dist": true
```

Comment:
Hmm, does this include dist in subdirectories?

Reply:
Good catch. Fixing this now.